### PR TITLE
restore REUSE compliance

### PR DESCRIPTION
**Pull request by fneum**
https://api.reuse.software/info/github.com/pypsa/atlite

### CODE REVIEW

File: .reuse/dep5 [hunk 1]
```diff
@@ -13,4 +13,16 @@ License: GPL-3.0-or-later
 
 Files: doc/img/*
 Copyright: 2019 The Atlite Authors
```
Comment (euronion) [inline .reuse/dep5:15]:
```suggestion
Copyright: 2019-2021 The Atlite Authors
```

File: .reuse/dep5 [hunk 2]
```diff
@@ -13,4 +13,16 @@ License: GPL-3.0-or-later
...
+Copyright: 2021 The Atlite Authors
+License: CC-BY-4.0
+
+Files: doc/examples/*.nblink
+Copyright: 2020-2021 The Atlite Authors
+License: CC-BY-4.0
+
+Files: .github/*
+Copyright: 2021 The Atlite Authors
+License: CC0-1.0
```
Comment (euronion) [inline .reuse/dep5:28]: `./reuse/dep5` is only intended for large directories or binary files (https://reuse.software/spec/#dep5).

Rather include the identifier into these files directly? For markdown that is
```
<!--
SPDX-FileCopyrightText: 2021 Jane

SPDX-License-Identifier: CC0-1.0
-->
```

or with the `reuse` tool
```
reuse addheader --copyright="2021 The Atlite Authors" --license="CC0-1.0" <file paths>
```

If you want I can do it.

File: README.rst [hunk 3]
```diff
@@ -6,7 +6,7 @@
 Atlite
 ======
 
-|PyPI version| |Conda version| |Documentation Status| |travis| |standard-readme compliant|
+|PyPI version| |Conda version| |Documentation Status| |travis| |standard-readme compliant| |GPL-3-or-later-image| |reuse|
```
Comment (euronion) [inline README.rst:9]: Is the GPL-3-or-later badge appropriate as we have multiple licenses for different aspects of the repo in place?

**Comment by euronion** [comment]
Thanks for the PR and license update!

File: README.rst (hunk 3 above)
Comment (fneum) [reply README.rst:9]: The main code is GPL-3 so I think it's ok.

File: .reuse/dep5 (hunk 2 above)
Comment (fneum) [reply .reuse/dep5]: Ok, done. I have kept it for the images and `*nblink` which I think are automatically generated.

`reuse>=0.13` also supports `*.ipynb` files with `reuse addheader ...` which creates a `.license` file.

Comment (euronion) [reply .reuse/dep5]: The `*.nblink` files are JSONs manually created by us and used by the SPHINX extension `nbsphinx_link`. So you could also add the header to those.

Great to hear REUSE finally supports `*.ipynb`'s :)

Comment (euronion) [reply .reuse/dep5]: Although I think the `*.ipynb.license` structure is clear than adding the license to the `./reuse/dep5` file =/

Comment (fneum) [reply .reuse/dep5]: Do JSONs have comments? probably it would work out as `*.nblink.license`

Comment (euronion) [reply .reuse/dep5]: No, JSONs don't have comments. Will probably generate the `.license` file. So better leave it in `dep5` then :)

**Comment by euronion** [comment]
Agreed. Thank you!

**Pull request merged**